docs/vivo-basic-usage.md

```diff
@@ -0,0 +1,276 @@
+
+# Vivo Basic Usage
+- [What is the value of Vivo?](#what-is-the-value-of-vivo)
+  - [User Interface Description](#user-interface-description)
+    - [Searching Logs, Metrics and Traces](#searching-logs-metrics-and-traces) 
+- [How can I start using it?](#how-can-i-start-using-it)
+  - [Using Docker](#using-docker)
+  - [Using Kubernetes](#using-kubernetes)
+- [Where is my data sent over?](#where-is-my-data-sent-over)
+
+## What is the value of Vivo?
+Calytptia Vivo is a live viewer for Fluent Bit streams. It provides a visualization interface for logs, metrics, and traces. It supports the work to get insights into the behavior and performance of your software system by helping in understanding what is happening within your applications and infrastructure, diagnose issues, and optimize performance, allowing you to explore the events that occurred within a set of components and track the occurrence of an event to act on.
+
+### User Interface Description
+Its User Interface (UI) has a sidebar with three buttons for each telemetry data type: Logs, Metrics, and Traces. You can click on any of such controls and get the data displayed for each data type in the central pane. Once the data is displayed, browse it using your mouse scroll button.
+
+![Img 1](img/Image_1.png)
+
+
+
+If you require inspecting a specific line, you can expand it by clicking the label with the “Open” legend that appears at the right-most place of the line you are located at.
+
+For example:
+![Img 2](img/Image_2.png)
+
+![Img 3](img/Image_3.png)
+
+
+#### Searching Logs, Metrics and Traces
+
+Through the UI, you can search logs, metrics, and traces by typing your search term in the search boxes of the UI. You can search logs by metadata, event, and both. The metrics and traces can be searched by event.
+
+For example:
+
+Searching logs  by Metadata:
+
+![Img 4](img/Image_4.png)
+
+Searching by Event
+
+![Img 5](img/Image_5.png)
+
+Searching by  Metadata and Event
+
+![Img 6](img/Image_6.png)
+
+
+## How can I start using it
+A quick way to start using Vivo is by following the next steps.
+
+### Using Docker
+
+#### 1.- Execute Vivo by using the  docker run command
+You can execute Vivo using the docker run command to start a Docker container based on the “calyptia/vivo" image, forwarding ports 8000 and 9000 from your host machine to ports 3000 and 24224 inside the container, respectively. The “vivo” application inside the container will be executed as the main process. Once the container finishes its task, it will be automatically removed ( because of the –rm flag).
```

Comment:
```suggestion
You can execute Vivo using the docker run command to start a Docker container based on the “calyptia/vivo" image, forwarding ports 8000 and 9000 from your host machine to ports 3000 and 24224 inside the container, respectively. The “vivo” application inside the container will be executed as the main process. Once the container finishes its task, it will be automatically removed (because of the –rm flag).
```